### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by CripyIce** [java-james/loading_overlay#4]
This library don't seem to work at all. No errors though.
Flutter doctor:
```
[✓] Flutter (Channel beta, v1.14.6, on Mac OS X 10.15.3 19D76, locale en-IL)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 3.6)
[!] VS Code (version 1.42.1)
    ✗ Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter
[✓] Connected device (3 available)
```

**Comment by java-james** [comment]
Hey thanks for the issue report.
Any more details on this? I'm consuming on `1.12.13+hotfix.8` without issues. 
Any specific error being thrown at runtime? Or you getting no overlay displayed when using it?
What does the code you've implemented it in look like?

**Comment by CripyIce** [comment]
Hi @java-james , there are no errors.
This is my code - very simple:
```dart
void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: MyHomePage(),
    );
  }
}

class MyHomePage extends StatefulWidget {
  MyHomePage({Key key}) : super(key: key);

  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text("Overlay Test"),
      ),
      body: LoadingOverlay(
        isLoading: true,
        child: Column(
          crossAxisAlignment: CrossAxisAlignment.start,
          children: <Widget>[
            Text(
              'Some text',
            ),
          ],
        ),
      ),
    );
  }
}
```

**Comment by aldwnesx** [comment]
not working as well

**Comment by aldwnesx** [comment]
> Hey thanks for the issue report.
> Any more details on this? I'm consuming on `1.12.13+hotfix.8` without issues.
> Any specific error being thrown at runtime? Or you getting no overlay displayed when using it?
> What does the code you've implemented it in look like?

1.12.13 doesn't exist?

**Comment by java-james** [comment]
> > Hey thanks for the issue report.
> > Any more details on this? I'm consuming on `1.12.13+hotfix.8` without issues.
> > Any specific error being thrown at runtime? Or you getting no overlay displayed when using it?
> > What does the code you've implemented it in look like?
> 
> 1.12.13 doesn't exist?

Ah referring to the flutter version I'm running on. Thanks I will use the info you've provided to reproduce the issue and report back here.

**Comment by kashlo** [comment]
same, doesnt work when LoadingOverlay is child of Scaffold

**Comment by java-james** [comment]
Have had a look into this. Seems like there is an issue initializing isLoading as true. (It's not picked up in the init state of the loading overlay).
Can get working fine with your layout when initializing as false then changing to true via state.
Will push this fix and publish this week.

<img width="415" alt="Screen Shot 2020-03-22 at 10 52 47 PM" src="https://user-images.githubusercontent.com/22756998/77246830-dbaea380-6c8f-11ea-8b3a-17e147a0abf9.png">

**Comment by java-james** [comment]
Have pushed the update as 0.2.1
Please try this version and confirm the issue is fixed
James

**Comment by java-james** [comment]
Closing this issue. Fixed in 0.2.1 let me know if you have any other issues
